Main/Include/char.h

```diff
@@ -1249,6 +1251,7 @@ class character : public entity, public id
   v2 GoingTo;
   double RandomMoveDir;
   long Money;
+  int MyVomitMaterial;
```

Comment:
Need to update `Load()` and `Save()`. Otherwise this variable will be undefined after loading the save file. I've encountered a few vomit crashes already. Anyway the old saves will be invalidated.

Reply:
Oops, thanks @iology!